OC-855 : Minor documentation updates (#274)
opfab/operatorfabric-core: src/docs/asciidoc/reference_doc/card_structure.adoc

ifndef::single-page-doc[<<{gradle-rootdir}/documentation/current/reference_doc/index.adoc#display_rules, Display rules>>]
. Without recipient declaration a card is useless in `OperatorFabric` system.

=== Card Life Management Configuration

> With this attributes OperatorFabric know when to display or hide cards.

Reviewer: ```suggestion
With this attributes OperatorFabric knows when to display or hide cards.
```

ifndef::single-page-doc[<<{gradle-rootdir}/documentation/current/reference_doc/index.adoc#display_rules, Display rules>>]
for some examples.

=== Optional information

==== Tags (`tag`)

> Tags are intended to enable filter of cards in the feed of the GUI.

Reviewer: ```suggestion
Tags are intended as an additional way to filter cards in the feed of the GUI.
```

==== EntityRecipients (`entityRecipients`)

> Use to send to cards to entity : all users members of the entities listed will receive the card. If it is used in conjonction with groups recipients, users must be member of one of the entities AND one of the groups to received the cards.

Reviewer: ```suggestion
Used to send to cards to entity : all users members of the listed entities will receive the card. If it is used in conjunction with groups recipients, users must be members of one of the entities AND one of the groups to receive the cards.
```

=== Last Time to Decide (`lttd`)

Fixes the moment until when a `actions` associated to the card are available. After then, the associated actions won't be displayed or actionable.

=== Store information 

==== uid (`uid`)

> Unique identifier of the card in the OperatorFabric system. This attribute can be send with card, but normally it's manage by `OperatorFabric` system.

Reviewer: ```suggestion
Unique identifier of the card in the OperatorFabric system. This attribute can be sent with card, but by default it's managed by OperatorFabric.
```

When the simple startDate and endDate are not enough to characterize your
process business times, you can add a list of TimeSpan to your card. TimeSpans

> are rendered in the timeline component as cluster bubbles. This as no effect on the feed

Reviewer: ```suggestion
are rendered in the timeline component as cluster bubbles. This has no effect on the feed
```

In this sample, the card will be displayed twice in the time line. The card
start date will be ignored.

> For timeSpans, you can specify an end date but it is not used by operator fabric (it was intended for future uses but it will be deprecated)

Reviewer: ```suggestion
For timeSpans, you can specify an end date but it is not implemented in OperatorFabric (it was intended for future uses but it will be deprecated).
```
